src/test/python/images/pyconv.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+
+import sys
+import csv
+import numpy as np
+from scipy.misc import imread, imsave
+from scipy.signal import convolve
+
+#This script convolves an image in python and its output is used in
+#the Convolver unit tests to ensure that convolver output matches
+#an equivalent python call.
+
+def main():
```

Comment:
Hmm that makes sense. Could you just put a comment saying that you already executed it (and the command you used)?